src/functionalTest/resources/features/F-001/F-001_Test_Base_data.td.json

```diff
@@ -0,0 +1,97 @@
+{
+  "_guid_": "F-460_Test_Data_Base_Get_Dcoument",
+  "productName": "CCD Data Store",
+  "operationName": "Get Case for Case Worker",
+  "specs": [
+    "Get the document from Document Store"
+  ],
+  "method": "GET",
+  "uri": "/documents/{documentId}",
+  "user": {
+    "username": "befta.caseworker.2.solicitor.3@gmail.com",
+    "password": "Pa55word11"
+  },
+  "request": {
+    "headers": {
+      "_extends_": "Common_Request_Headers"
+    },
+    "pathVariables": {
+      "documentId": "fcdca3b6-607f-41af-ad8c-92267c921d44"
+    }
+  },
+  "expectedResponse": {
+    "responseCode" : 200,
+    "responseMessage" : "OK",
+    "headers": {
+      "Access-Control-Allow-Credentials" : "true",
+      "Access-Control-Allow-Headers" : "undefined",
+      "Access-Control-Allow-Methods" : "GET,POST,OPTIONS,PUT,DELETE",
+      "Cache-Control" : "[[ANY]]",
+      "connection" : "[[ANY]]",
+      "content-type" : "[[ANY]]",
+      "date" : "[[ANY]]",
+      "Expires" : "0",
+      "Pragma" : "no-cache",
+      "request-context" : "[[ANY]]",
+      "Strict-Transport-Security" : "[[ANY]]",
+      "Surrogate-Control" : "no-store",
+      "transfer-encoding" : "chunked",
+      "x-content-type-options" : "nosniff",
+      "x-frame-options" : "DENY",
+      "x-xss-protection" : "1; mode=block"
+    },
+    "body" : {
+      "size" : 164215,
+      "mimeType" : "image/png",
+      "originalDocumentName" : "Screenshot 2019-12-30 at 16.49.21.png",
+      "createdBy" : "a54ba0bd-6d8b-46ab-9a1e-87cfeebaefdd",
+      "lastModifiedBy" : "a54ba0bd-6d8b-46ab-9a1e-87cfeebaefdd",
+      "modifiedOn" : "[[ANY]]",
+      "createdOn" : "[[ANY]]",
+      "classification" : "PUBLIC",
+      "roles" : [ ],
+      "metadata" : { },
+      "_links" : {
+        "self" : {
+          "href" : "http://dm-store:8080/documents/ac3ada44-1a57-42d4-8313-07067c6f50ff"
+        },
+        "binary" : {
+          "href" : "http://dm-store:8080/documents/ac3ada44-1a57-42d4-8313-07067c6f50ff/binary"
+        },
```

Comment:
I wonder how come we can know these UUID values in an environment. These are generated UUIDs different on all envs.